lib/Perl/MinimumVersion.pm

```diff
@@ -870,6 +871,49 @@ sub _get_resulting_sigil {
 	}
 }
 
+# Postfix dereference new (and experimental) in 5.20, mainstream in 5.24.
+# THIS CODE ASSUMES PPI 1.237_001 OR ABOVE -- i.e. support for postfix
```

Comment:
The minimum of `1.237_001` is still correct and all production versions of PPI from that version to the latest work.

Tested with:
```
curl https://fastapi.metacpan.org/source/OALDERS/PPI-1.276/Changes | grep -P '^\d' | awk '$1 >= 1.236 {print$1}' | tac | xargs -n1 -I{} bash -c 'cpm install -g PPI@{} && prove -l t/27_postfix_deref.t'
```

Reply:
That minimum version should also be added to the `dist.ini` (new authoring set up since the PR was created) when rebasing.